### ISSUE TITLE
cors error in chrome

### ISSUE THREAD

**Issue by realcodywburns** [ethereumproject/explorer#73]
Cannot display some pages because:
`Failed to load http://127.0.0.1:3000/stats: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://localhost:3000' is therefore not allowed access.`

**Comment by realcodywburns** [comment]
`http://drawpie.com/etc_hash_rate_api` seems to be the culprit 

@elaine , can we math out the implied network hash for each block based on its blocktime and difficulty?

**Comment by elaineo** [comment]
yeah, it's better to do this all on the same server.... cc: @kimisan

**Comment by kimisan** [comment]
ok, i will fix it and update node.js script by this week.

**Comment by kimisan** [comment]
update some info, since i update the new Geth.
The DB has just synced 50%, sorry for delay.

**Comment by DialogueSolutions** [comment]
Is (http://drawpie.com/etc_hash_rate_api) supposed to help draw w3.js pie charts? (http://bl.ocks.org/juan-cb/1984c7f2b446fffeedde)

Just trying to see if there has been any update since it seems like the link is broken. :)

**Comment by mikeyb** [comment]
late to the party, but this chrome plugin `Moesif CORS` is what I use when developing locally and need to bypass the localhost cors issues

**Comment by hackmod** [comment]
currently, all chart related datas are generated on the fly.
